docs/guides/serverless-manage-resources.ipynb

```diff
@@ -73,18 +153,115 @@
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": 5,
    "id": "ac99b4a0-4a42-4c43-869d-265344b70359",
    "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Appending to ./source_files/transpile_remote.py\n"
+     ]
+    }
+   ],
    "source": [
-    "```python\n",
+    "%%writefile --append ./source_files/transpile_remote.py\n",
+    "\n",
     "transpile_worker_references = [\n",
     "    transpile_remote(circuit, optimization_level, backend)\n",
-    "    for circuit in circuit_list\n",
+    "    for circuit in arguments.get(\"circuit_list\")\n",
     "]\n",
     "\n",
     "results = get(transpile_worker_references)\n",
-    "```"
+    "save_result(results)  # Overwrites any previously saved results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "b872a4b0-280a-41e8-8fa3-6a633ea64b2e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Appending to ./source_files/transpile_remote.py\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%writefile --append ./source_files/transpile_remote.py\n",
+    "# This cell is hidden from users. It checks the transpilation ran correctly.\n",
+    "assert isinstance(results[0], QuantumCircuit)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "ad1914fa-bfd0-4aa0-936b-fa41bde6090e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Job still running, waiting 5s then retrying (19)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# This cell is hidden from users.\n",
+    "# It uploads the serverless program and checks it runs.\n",
+    "\n",
+    "from qiskit.circuit.random import random_circuit\n",
+    "from qiskit_serverless import IBMServerlessClient, QiskitFunction\n",
+    "from IPython.display import clear_output\n",
+    "import time\n",
+    "import uuid\n",
+    "\n",
+    "TITLE = \"transpile_remote_serverless_test_\" + uuid.uuid4().hex[:8]\n",
+    "serverless = IBMServerlessClient()\n",
+    "transpile_remote_demo = QiskitFunction(\n",
+    "    title=TITLE,\n",
+    "    entrypoint=\"transpile_remote.py\",\n",
+    "    working_dir=\"./source_files/\",\n",
+    "\tdependencies=[\"qiskit-aer==0.14.1\"],\n",
```

Comment:
Why is this hardcoded? I'm not very familiar with serverless

Reply:
I think it's just to demonstrate that you can set dependencies for your serverless code

Reply:
Hm, I wonder if we can remove it. I'm worried about the dep falling out of sync. Think about how many people copy and paste code, or even Qiskit Code Assistant learning from our docs. We don't want them installing an old version.

Reply:
lol I'm sorry; I didn't realise this was part of the testing code I added. I copied it from https://docs.quantum.ibm.com/guides/serverless-first-program. I'll remove the dpe.

> Think about how many people copy and paste code

👀